showup/migrations/0005_merge_20191118_2327.py

```diff
@@ -0,0 +1,13 @@
+# Generated by Django 2.2.5 on 2019-11-19 04:27
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("showup", "0004_merge_20191118_1643"),
+        ("showup", "0003_auto_20191118_1915"),
+    ]
+
+    operations = []
```

Comment:
What does this migration do? I suspect that it does nothing because "operations" is empty, but I'm not sure.